### PR TITLE
Added an IngredientStorageController

### PR DESCRIPTION
**Pull request by kailashs2002**
Need to repeat this process for a RecipeController (and MealPlanController/ShoppingListController)

### CODE REVIEW

**Comment by kailashs2002** [comment]
Stuff is breaking, closing

File: app/src/main/java/com/androidimpact/app/Ingredient.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
  * @version 1.0
  */
 public class Ingredient implements Serializable  {
+    protected String id;
```
Comment (joshuanianji) [inline app/src/main/java/com/androidimpact/app/Ingredient.java:23]: For an ID attribute, can you add a `@DocumentId` tag on top? It helps Firebase know how to serialize ingredients when reading/writing from firestore.

I think something like 

```
public class Ingredient implements Serializable  {
    @DocumentID
    protected String id;
    protected String description;
    protected float amount;
```

**Review by joshuanianji — COMMENTED** [review]
Everything looks good to me! adding document ID annotation is the only thing I can find that i would change

File: app/src/main/java/com/androidimpact/app/Ingredient.java (hunk 1 above)
Comment (joshuanianji) [reply app/src/main/java/com/androidimpact/app/Ingredient.java:25]: Ok sike, easy fix and not a big deal so I just changed it

**Review by joshuanianji — APPROVED** [review]

**Comment by joshuanianji** [comment]
Note this is related to #72

**Pull request merged**